torch/onnx/symbolic.py

```diff
@@ -1710,3 +1710,24 @@ def argmin(g, input, dim, keepdim):
         dim = _parse_arg(dim, 'i')
         keepdim = _parse_arg(keepdim, 'i')
         return g.op('ArgMin', input, axis_i=dim, keepdims_i=keepdim)
+
+
+@parse_args('v')
```

Comment:
i think no need to add `parse_args` in these cases (i.e., all inputs are tensors): https://github.com/pytorch/pytorch/blob/c484cf43a02863efd2f4a76aad43246fb0191ab5/torch/onnx/symbolic.py#L114

Reply:
done